tikv-client/src/main/java/com/pingcap/tikv/operation/iterator/DAGIterator.java

```diff
@@ -203,6 +202,8 @@ private SelectResponse process(RangeSplitter.RegionTask regionTask) {
     HashSet<Long> resolvedLocks = new HashSet<>();
     // In case of one region task spilt into several others, we ues a queue to properly handle all
     // the remaining tasks.
+    // Do we need to set the timeout for this loop? If all the stores are suspended, it will not
```

Comment:
What's the conclusion

Reply:
we can open another pr to solve if we need. This problem has always existed before, not introduced by this pr.